packages/js/tests/helpers/getValuesFromCheckboxes.test.js

```diff
@@ -0,0 +1,22 @@
+import { getValuesFromCheckboxes } from "../../src/helpers/getValuesFromCheckboxes";
+import { getPostCategoryCheckboxes } from "../../src/classic-editor/helpers/dom";
+import { createCategoryElements } from "../testHelpers/createDomTestData";
+
+const checkboxesElements = createCategoryElements();
+document.body.appendChild( checkboxesElements.allCategories );
+
+describe( "a test for retrieving the values from checked checkboxes", () => {
+	it( "should return an empty array if no checkboxes is checked", () => {
+		const checkedCheckboxes = getPostCategoryCheckboxes().filter( checkbox => checkbox.checked );
+		expect( getValuesFromCheckboxes( checkedCheckboxes ) ).toEqual( [] );
+	} );
```

Comment:
I'm not sure if we want this test to depend on the `getPostCategoryCheckboxes` function. 

If this test fails, we do not know if it is because of a failure of `getValuesFromCheckboxes`, or a failure in `getPostCategoryCheckboxes`.